src/main/java/ca/corefacility/bioinformatics/irida/security/OauthRedirectResolver.java

```diff
@@ -0,0 +1,30 @@
+package ca.corefacility.bioinformatics.irida.security;
+
+import org.springframework.security.oauth2.common.exceptions.InvalidRequestException;
+import org.springframework.security.oauth2.common.exceptions.OAuth2Exception;
+import org.springframework.security.oauth2.provider.ClientDetails;
+import org.springframework.security.oauth2.provider.endpoint.DefaultRedirectResolver;
+
+/**
+ * Redirect resolver for Oauth2 clients that allows any url for redirection
+ */
+public class OauthRedirectResolver extends DefaultRedirectResolver {
```

Comment:
We should mark this as a temporary fix while we're doing development.  We should have the redirect URIs registered on client creation.  Can you maybe `@Depreciated` this so we don't forget?

Reply:
Added in f82f501296f49ac938eae69f6de8dda70d2daadc